### DISCUSSION TITLE
Capstone Project, WebScrapping, Error saat ubah tipe data

### DISCUSSION THREAD

**Discussion by tu849035** [teamalgoritma/community#200] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/107348211/178189101-c1b27a69-824e-470c-9652-b60761d4fd95.png)

Mohon bantuannya, saat akan mengubah tipedata seperti tampilan di atas, muncul error.

Terima Kasih.

**Comment by triarm** [comment, marked as answer]
Hi @tu849035 ,
untuk mengubah tipe data kolom Volume menjadi float maka Anda perlu membersihkan tanda "$" dan "," terlebih dahulu. Anda dapat mencoba kode berikut untuk membersihkan data beserta mengubah tipe datanya.

```{python}
df['Volume'] = df['Volume'].str.replace("$", "")
df['Volume'] = df['Volume'].str.replace(",", "")
df['Volume'] = df['Volume'].astype('float64')
```

Demikian yang dapat kami sampaikan. Semoga membantu

**Reply by tu849035** [reply]
![image](https://user-images.githubusercontent.com/107348211/178190561-c8389629-6a51-486d-b2d2-37a52993b5c9.png)
![image](https://user-images.githubusercontent.com/107348211/178190798-e35633b4-5f4c-4dc2-865d-f43477e434b0.png)
![image](https://user-images.githubusercontent.com/107348211/178190839-f43abc5d-7815-4d61-a6ee-1731c374ad86.png)

Sudah saya coba seperti kode di atas, tp masih muncul errornya.
Mohon bantuannya.

**Reply by triarm** [reply]
Hal tersebut terjadi karena masih terdapat tanda titik (.) pada kolom Volume. Untuk mengatasi kendala tersebut, silahkan untuk menambahkan kode berikut sebelum mengubah tipe data
```
df['Volume'] = df['Volume'].str.replace(".", "")
```
